src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -97,23 +140,24 @@ public void run() {
     }
   }
 
-  private final IndexCollectionArgs args;
+  private final IndexCollection.Args args;
   private final Path indexPath;
   private final Path collectionPath;
   private final Class collectionClass;
   private final Class transformerClass;
   private final Collection collection;
   private final Counters counters;
 
-  public IndexCollection(IndexCollectionArgs args) throws Exception {
+  public IndexCollection(IndexCollection.Args args) throws Exception {
     this.args = args;
 
     LOG.info("Index path: " + args.index);
     LOG.info("Threads: " + args.threads);
     LOG.info("Keep stopwords: " + args.keepstop);
     LOG.info("Positions: " + args.positions);
     LOG.info("Store docvectors: " + args.docvectors);
-    LOG.info("Store docs: " + args.storeTransformedDocs);
+    LOG.info("Store docs: " + (args.storeTransformedDocs ? "transformed" :
+        (args.storeRawDocs ? "raw" : "none")));
```

Comment:
if `storeTransformedDocs` and `storeRawDocs` are mutually exclusive, shouldn't this be explained in the Arguments description?